features/testbot/via.feature

```diff
@@ -18,6 +18,23 @@ Feature: Via points
             | waypoints | route           |
             | a,b,c     | abc,abc,abc,abc |
 
+    Scenario: Simple via point with waypoints collapsing
+        Given the node map
+            """
+            a b c
+            """
+
+        And the ways
+            | nodes |
+            | abc   |
+
+       Given the query options
+            | waypoints | 0;2   |
+
+        When I route I should get
+            | waypoints | route       |
+            | a,b,c     | abc,abc     |
```

Comment:
Let’s also test the case where the waypoint is located at an intersection and the route ends up making a non-obvious turn at that intersection; we’d expect the route to contain a maneuver at that point, rather than eliding it completely.